### DISCUSSION TITLE
urSim for mac users

### DISCUSSION THREAD

**Discussion by aovaaa** [visose/Robots#229] [Q&A] [answered]
I am working on a script using robots (Grasshopper) but when I wanted to download the URsim to run simulations I realized that the 2 virtual machines recommended do not support my Macbook M1 OS. So I was wondering what virtual machine mac users are using to run URsim?

I noticed that VirtualBox have a beta version for mac but it seams to be quite buggy

**Comment by robin-gdwl** [comment, marked as answer]
This is a general Problem with URsim currently. 
There may be solutions using docker described in [this forum post](https://forum.universal-robots.com/t/ursim-on-m1-macbook/14668/5?u=rodwyll).
Apart from that you can join the discussion on the Universal Robot Forum in these threads: 

https://forum.universal-robots.com/t/ursim-on-m1-macbook/14668
https://forum.universal-robots.com/t/how-to-install-polyscope-on-macbook-pro-m1/22959/6
